### PR TITLE
Fix (unified) transaction listing on PayPal.com

### PR DESCRIPTION
**Pull request by oberrich**
- Fixes unified transaction view on PayPal.com

image
*(before)*

image
*(after)*

### CODE REVIEW

**Comment by oberrich** [comment]
De-uglified `input`s, see refund view:

image
*(before)*

image
*(after)*

**Comment by alexanderby** [comment]
Could you please share the source CSS colors assigned there (with Dark Reader disabled)? The support for those color mixes was added recently, maybe it should be improved.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -19598,8 +19598,20 @@ img[src*="/img/bank-logos/"]
 paypal.com
 
 CSS
+input {
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:19601]: Is there a way to be more specific there (by referencing elements with class names for example).

**Comment by oberrich** [comment]
Dump of styles (devtools)
```css
.tab_container {
    line-height: 1.15;
    -webkit-text-size-adjust: 100%;
    --animate-duration: 1s;
    --animate-delay: 1s;
    --animate-repeat: 1;
    font-family: PayPalSansSmall-Regular,Helvetica Neue,Arial,sans-serif;
    -webkit-font-smoothing: antialiased;
    --darkreader-inline-outline: initial;
    box-sizing: inherit;
    background: var(--sys-background-surface,#fff);
    padding-bottom: 32px;
}
.tab_container:dark-reader {
    color-scheme: dark !important;
    line-height: 1.15;
    -webkit-text-size-adjust: 100%;
    --darkreader-neutral-background: #131516;
    --darkreader-neutral-text: #d8d4cf;
    --darkreader-selection-background: #004daa;
    --darkreader-selection-text: #e8e6e3;
    --animate-duration: 1s;
    --animate-delay: 1s;
    --animate-repeat: 1;
    color: #e8e6e3;
    -webkit-font-smoothing: antialiased;
    --darkreader-inline-outline: initial;
    box-sizing: inherit;
    font-family: Gadugi !important;
    background: var(--sys-background-surface,#fff);
    padding-bottom: 32px;
}

.listing_bg {
    line-height: 1.15;
    -webkit-text-size-adjust: 100%;
    --animate-duration: 1s;
    --animate-delay: 1s;
    --animate-repeat: 1;
    font-family: PayPalSansSmall-Regular,Helvetica Neue,Arial,sans-serif;
    -webkit-font-smoothing: antialiased;
    --darkreader-inline-outline: initial;
    box-sizing: inherit;
    background-color: #faf8f5;
    height: auto;
}
.listing_bg:dark-reader {
    color-scheme: dark !important;
    line-height: 1.15;
    -webkit-text-size-adjust: 100%;
    --darkreader-neutral-background: #131516;
    --darkreader-neutral-text: #d8d4cf;
    --darkreader-selection-background: #004daa;
    --darkreader-selection-text: #e8e6e3;
    --animate-duration: 1s;
    --animate-delay: 1s;
    --animate-repeat: 1;
    color: #e8e6e3;
    -webkit-font-smoothing: antialiased;
    --darkreader-inline-outline: initial;
    box-sizing: inherit;
    font-family: Gadugi !important;
    height: auto;
    background-color: rgb(40, 32, 20);
}
```

_`:dark-reader` isn't real and merely for distinction_

**Comment by alexanderby** [comment]
Ah, I thought the `color-mix()` was used there. You can try using `background-color: ${#faf8f5) !important;` instead to make it simpler.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (alexanderby) [reply src/config/dynamic-theme-fixes.config:19601]: I'm afraid that all the inputs will look different when not expected. Please use the selectors that correspond with website's own border styles.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -19598,8 +19598,20 @@ img[src*="/img/bank-logos/"]
...
+    border-style: solid !important;
+    border-width: 1px !important;
+}
 .cw_tile-promo-title-container_dw2.css-1cak51m-card_base {
-    background-image: none !important;
+    background: none !important;
+}
+.tab_container,
+.listing_bg {
+    background-color: color-mix(in srgb, var(--darkreader-neutral-text) 10%, var(--darkreader-neutral-background) 90%) !important;
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:19610]: Please try simplifying it with `background-color: ${#faf8f5) !important;`.

**Review by alexanderby — CHANGES_REQUESTED** [review]

**Review by alexanderby — APPROVED** [review]

**Pull request merged**